pkgs/top-level/coq-packages.nix

```diff
@@ -54,7 +54,8 @@ in rec {
 
   mkCoqPackages = coq:
     let self = mkCoqPackages' self coq; in
-    filterCoqPackages coq self;
+    if coq.doNotFilter or false then self
```

Comment:
```suggestion
    if coq.doNotFilter then self
```

Reply:
Right! It's the `a.b or c` syntax. I can't believe I fell for that.

Do we really need the negated name? Negated names require unnecessary extra thinking. What about "setting `doFilter = false`" which has a default of true?

Reply:
Maybe Théo has an opinion. Ping @Zimmi48 .

Reply:
Ah I did not know about the syntax, will use ;-)
Regarding `doFilter = false` vs `doNotFilter = true`, it seems that the implicit convention in nixpkgs (https://nixos.org/nixpkgs/manual/#sec-stdenv-phases) is that when something is off by default, it is enabled with `doStuff = true` but when something is on by default, it is disabled `dontStuff = true`. Thus, it suggests to use `dontFilter` here.

Reply:
Let’s go with `dontFilter`.